lib/jsonDir.js

```diff
@@ -32,8 +32,21 @@ var collectJSONS = function (options) {
         }
     }
 
+    function addTimestamp(featureItem) {
+        if (typeof featureItem["elements"][0]["steps"][0]["output"][0] !== 'undefined') {
```

Comment:
@lkotek I looked at the CI Build, and it's failing at this line. I'd suggest to deep-check each of the object.

```
/home/travis/build/gkushang/cucumber-html-reporter/lib/jsonDir.js:36
        if (typeof featureItem["elements"][0]["steps"][0]["output"][0] !== 'undefined') {
                                                                   ^
TypeError: Cannot read property '0' of undefined
```

Build: https://travis-ci.org/gkushang/cucumber-html-reporter/builds/610207394?utm_source=github_status&utm_medium=notification

Reply:
@gkushang Yep, I know. Problem is, that whole modification is based on usage of this element (which is not mandatory and not always present) containing timestamp. So unfortunately I can't see any way how to avoid this failure in CI. PR is written in a way, that even in situation this element doesn't exist, adding of timestamp is skipped and html reporter behaves exactly the same as before. Is it possible to modify CI to use `scenarioTimestamp: true`, please? If not I understand this can make this PR not acceptable then I see no other way than dropping this PR.

Reply:
Sure, never mind! Let me take a look at it

Reply:
Tests work fine by deep comparing `undefined` for each hierarchy ... 

I modified, `addTimestamp` like below - can you try at your side to see the feature you want to implement works fine with below changes? If so, the CI will accept the changes!

```
    function addTimestamp(featureItem) {
        if (featureItem["elements"] && featureItem["elements"][0] &&
            featureItem["elements"][0]["steps"] && featureItem["elements"][0]["steps"][0] &&
            featureItem["elements"][0]["steps"][0]["output"] && featureItem["elements"][0]["steps"][0]["output"][0]) {

            if (typeof featureItem["elements"][0]["steps"][0]["output"][0] !== 'undefined') {
                var timestamp = featureItem["elements"][0]["steps"][0]["output"][0];
                featureItem["timestamp"] = Date.parse(timestamp.match(/[0-9]{4}-.+:[0-9]{2}/g));
            }
        }
        return featureItem;
    }
```

Reply:
@gkushang Thank you a lot! I modified the way you proposed and both is fine :-)